log-insights/setup/aiven/04_configure_service_integration.mdx

```diff
@@ -25,14 +25,25 @@ The last step is setting up the Aiven Service Integration. Go to the
 Service Integration page in your Aiven console, select Syslog from the
 list of options, and click "Add new endpoint".
 
+<strong>Setup using VPC</strong>
```

Comment:
Not a big deal, but can we use a `### subheading` here? (Possible with more or less `#`s.)

Reply:
I found that these also-shown-in-app markdown files are not using markdown that much for some reason (well some of them are legit reasons, like images and links), but I think `###` is supported. I'll change this and see how it looks in both web and in app.

Reply:
yup, `###` worked fine:

In web:

![image](https://github.com/pganalyze/pganalyze-docs/assets/911433/1fe29a92-2eb5-45d8-9e90-ec0740869a7f)

In App:

![image](https://github.com/pganalyze/pganalyze-docs/assets/911433/f0f85a43-8f21-47f6-be0c-47411d674516)